src/commands/Tools/Dictionary/urban.ts

```diff
@@ -16,43 +17,54 @@ export default class extends SkyraCommand {
 			nsfw: true,
 			requiredPermissions: ['EMBED_LINKS'],
 			runIn: ['text'],
-			usage: '<query:string> [page:integer{0,10}]',
-			usageDelim: '#'
+			usage: '<query:string>'
 		});
 	}
 
-	public async run(message: KlasaMessage, [query, ind = 1]: [string, number]) {
-		const index = ind - 1;
-		if (index < 0) {
-			throw message.language.tget('RESOLVER_POSITIVE_AMOUNT');
-		}
+	public async run(message: KlasaMessage, [query]: [string]) {
+		const response = await message.sendEmbed(
+			new MessageEmbed({ description: message.language.tget('SYSTEM_LOADING'), color: getColor(message) || 0xFFAB2D })
```

Comment:
Always prefer constructor arguments:

```suggestion
			new MessageEmbed()
				.setDescription(message.language.tget('SYSTEM_LOADING'))
				.setColor(getColor(message) || 0xFFAB2D)
```